freqtrade/data/dataprovider.py

```diff
@@ -116,3 +116,17 @@ def runmode(self) -> RunMode:
         can be "live", "dry-run", "backtest", "edgecli", "hyperopt" or "other".
         """
         return RunMode(self._config.get('runmode', RunMode.OTHER))
+
+    def current_whitelist(self) -> List[str]:
+        """
+        fetch latest available whitelist.
+
+        Useful when you have a large whitelist and need to call each pair as an informative pair.
+        As available pairs does not show whitelist until after informative pairs have been cached.
+        :return: list of pairs in whitelist
+        """
+        from freqtrade.pairlist.pairlistmanager import PairListManager
+
+        pairlists = PairListManager(self._exchange, self._config)
+        pairlists.refresh_pairlist()
+        return pairlists.whitelist
```

Comment:
i don't think this is a good way of implementing this.

It's creating a second instance of PairListManager. 
That's not a problem per se - but PairListManager does some caching (of tickers, for example) - and also of the VolumePairlist - so if you're not calling this in the same frequence you're calling the real pairlist, the 2 can become misaligned - so you get a pairlist - but not necessarily the same one the bot is using to create trades.
Also, it's adding additional network calls (to tickers, but to anything in theory assuming additonal pairlists are implemented) - so i would avoid having 2 instances around all the time.

My proposal for this woul'd be:
add it as a 3rd, (optional?) argument to `__init__()` - and keep the instance passed in as `self._pairlists` (or something similar).

That way, you won't have to call `refresh_pairlist()` but can rely on the call done by freqtradebot.
You can then check in this method if that exists or is None - and eventually return (or raise an Exception?) if dataprovider was not initialized with a pairlistprovider.